second-edition/src/ch20-06-graceful-shutdown-and-cleanup.md

```diff
@@ -36,13 +26,9 @@ impl Drop for ThreadPool {
 <span class="caption">Listing 20-22: Joining each thread when the thread pool
 goes out of scope</span>
 
-We loop through each of the thread pool `workers`, using `&mut` because `self`
-is itself a mutable reference and we also need to be able to mutate `worker`.
-We print out a message saying that this particular worker is shutting down, and
-then we call `join` on that worker's thread. If the call to `join` fails, we
-`unwrap` the error to panic and go into an ungraceful shutdown.
+Recorrimos cada uno de los `pool` del grupo de subprocesos, usando` & mut` porque `self` es en sí misma es una referencia mutable y también necesitamos poder mutar `worker`. Imprimimos un mensaje que dice que este worker en particular se está cerrando, y luego llamamos a `join` al hilo de ese worker. Si la llamada a `join` falla, nosotros `deploy` el error para entrar en pánico y realizar un apagado desagradable.
```

Comment:
Would recommend a different approach for 'apagado desagradable', maybe 'cierre elegante', but is a personal opinion, wdyt?